krux_cloud_health/cloud_health.py

```diff
@@ -0,0 +1,140 @@
+# -*- coding: utf-8 -*-
+#
+# © 2016 Krux Digital, Inc.
+#
+"""
+Class to retrieve data from Cloud Health Tech API
+"""
+
+#
+# Standard libraries
+#
+
+from __future__ import absolute_import
+import sys
+import urlparse
+
+#
+# Third party libraries
+#
+
+import requests
+
+#
+# Internal libraries
+#
+
+from krux.cli import Application
+from krux.cli import get_parser, get_group
+
+API_ENDPOINT = "https://apps.cloudhealthtech.com/"
+NAME = "cloud-health-tech"
+
+
+def add_cloud_health_cli_arguments(parser):
+    # Add those specific to the application
+    group = get_group(parser, NAME)
+    
+    group.add_argument(
+        'api_key',
+        type=str,
+        help="API key to retrieve data",
+    )
+
+
+def get_cloud_health(args, logger, stats):
+    if not args:
+        parser = get_parser(description=NAME)
+        add_dns_cli_arguments(parser)
+        args = parser.parse_args()
+
+    if not logger:
+        logger = get_logger(name=NAME)
+
+    if not stats:
+        stats = get_stats(prefix=NAME)
+
+    return CloudHealth(
+        api_key=args.api_key,
+        logger=logger,
+        stats=stats,
+        )
+
+
+class CloudHealth(object):
+    """
+    """
+    def __init__(self, api_key, logger, stats):
+        self.api_key = api_key
+        self.logger = logger
+        self.stats = stats
+
+    def costHistory(self, month_input=None):
+        report = "olap_reports/cost/history"
+        api_call = self.get_api_call(report, self.api_key)
+
+        months_dict = api_call["dimensions"][0]["time"]
+        months_list = [str(month["name"]) for month in months_dict]
+
+        items_list = api_call["dimensions"][1]["AWS-Service-Category"]
+
+        if month_input is None:
+            total_data = []
+            for month_index in range(len(months_list)):
+                month = months_list[month_index]
+                month_info = self.get_month_info(api_call, items_list, month, month_index)
+                total_data.append(month_info)
+            return total_data
+
+        if month_input not in months_list:
+            raise ValueError("Invalid month input.")
+            self.exit(1)
+
+        month_index = months_list.index(month_input)
+        month_info = self.get_month_info(api_call, items_list, month_input, month_index)
+        return month_info
+
+    def get_month_info(self, api_call, items_list, month, month_index):
```

Comment:
Is this intended to be a public function? If this is a private helper, prefix it with an underscore: `_get_month_info()`.